CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ option(
 option(VELOX_MONO_LIBRARY "Build single unified library." OFF)
 option(ENABLE_ALL_WARNINGS "Enable -Wall and -Wextra compiler warnings." ON)
 option(VELOX_BUILD_SHARED "Build Velox as shared libraries." OFF)
+option(VELOX_BUILD_BRANCH_KERNEL_TEST "Enable Wave branch kernel test." ON)
```

Comment:
Can we define it as an opt-out instead of opt-in?  Something like `VELOX_SKIP_WAVE_BRANCH_KERNEL_TEST`

Reply:
Yes, done in latest version of PR.